### PR TITLE
Feature/text input

### PR DESCRIPTION
**Pull request by gjtrecny**
# Description

Please check the type of change your PR introduces:
- [X] Bugfix (change which fixes an issue)
- [X] Feature (change which adds functionality)
- [X] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

 
Created a text area for the user to write in in the TextInput Component. There were some errors related to defining an entry as an object. May cause some issues down the line since a single story does need to be an object. 

 

### Type of change

Please check the relevant option.

- [ ] Non-Breaking Change
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Big Breaking change (this breaks everything and I am sorry)

# Where should reviewers start?

# How Has This Been Tested?

Please check the option(s) that best describe your tests. 

- [ ] New spec files were added
- [ ] Tests were added to existing spec files
- [ ] Some tests were changed
- [ ] No tests were changed
- [ ] All tests were changed (please explain what in God's name happened)

### Tests Description

- Please add further details of your tests here.

# Checklist:

- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have removed all unused code or commented why I left it
- [ ] I have deployed my changes to a test-heroku and they work
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass locally with my changes

### CODE REVIEW

File: src/Components/TextInput/TextInput.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    color: white;
+    width: 20%;
+    margin: 3%;
+    padding: 2%;
+    box-shadow: none;
+    border: none;
+    border-radius: 10px;
+}
+
+@media (max-width: 500px) {
```
Comment (Alynn022) [inline src/Components/TextInput/TextInput.scss:19]: Love that you're doing mobile first! I need to follow your footsteps!

**Review by Alynn022 — APPROVED** [review]

File: src/Components/TextInput/TextInput.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import './TextInput.scss';
+
+const TextInput: React.FC = () => {
+  const [ entry, setEntry ] = useState<string>()
+
+  //Place for posting the writing entry once completed
+  //Did not inpout any actual functionality until we know our data
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault()
+    console.log(entry)
```
Comment (philalewis) [inline src/Components/TextInput/TextInput.tsx:10]: We'll have to pass down a function to add this to the post request body.

File: src/Components/TextInput/TextInput.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+  
+  return (
+    <>
+      <form>
+        <textarea
+          aria-label='textarea'
+          placeholder='Begin writing here...'
+          rows={20}
+          cols={30}
```
Comment (philalewis) [inline src/Components/TextInput/TextInput.tsx:21]: I would love to know what is going on here. I haven't used rows and columns like this before.

**Review by philalewis — APPROVED** [review]

**Review by Jayquintana — APPROVED** [review]
looks great !

**Review by Alynn022 — APPROVED** [review]

**Pull request merged**